WebRoot/css/zm.css

```diff
@@ -6239,27 +6232,24 @@ INPUT[type="text"].ZmFolderPolicyAmountInput {
 	border-color:@SelC@ @darken(SelC,25)@ @darken(SelC,25)@ @SelC@;
 }
 
-
-DIV.ZmSuggestLink {
-	display:inline-block;
-	padding-right:8px;
-}
-
-
 .ZmLocationStatusValidating,
 .ZmLocationStatusConflict,
 .ZmLocationStatusResolved {
-	display:inline-block;
-	padding-right:8px;
+	@ApptEditLocationStatusMsgWrapper@
 }
-.ZmLocationStatusValidating {
-	color:black;
+
+.ZmLocationStatusValidating div[class*="Img"],
```

Comment:
Should we also have ends with selector here to make scope of this selector specific ?
div[class*="Img"][class$="svg"]

Reply:
I think, it's not needed, this will be enough.
Do you find any cases where we may need to have more specific selector?

Thanks

Reply:
I think if there's an image thats not available as SVG the element would have the class applied directly as well . 
Like ImgActive instead of ImgActive-svg which we might not need here. ?

Reply:
Even for that we need similar styles. So I see no problem here. If you see rule, it just sets it to `inline-block` and has some spacing.